src/fermilib/utils/_parallel_bubble_sort.py

```diff
@@ -0,0 +1,114 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""This module has functions to perform parallel bubble sort on arrays."""
```

Comment:
I think you should have a space after the 1-line description and then have a block quote that gives a few sentences about why a parallel bubble sort code might be in FermiLib at all. Then put a space before the imports.